### ISSUE TITLE
"Cannot read property 'replace' of undefined" ???

### ISSUE THREAD

**Issue by csulit** [ozkanonur/nestjs-rate-limiter#67]
I'm Confuse about this error.

```
[Nest] 8284   - 30/04/2021, 2:15:21 pm   [ExceptionsHandler] Cannot read property 'replace' of undefined +27993ms
TypeError: Cannot read property 'replace' of undefined
    at RateLimiterInterceptor.<anonymous> (C:\Users\ChristianAngeloSulit\Documents\MHS-Final\mhs-nestjs-api\node_modules\nestjs-rate-limiter\dist\rate-limiter.interceptor.js:176:36)
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\ChristianAngeloSulit\Documents\MHS-Final\mhs-nestjs-api\node_modules\nestjs-rate-limiter\dist\rate-limiter.interceptor.js:17:58)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

**Comment by ozkanonur** [comment]
will be fixed on next version with adding an optional chaining operator at [here](https://github.com/ozkanonur/nestjs-rate-limiter/blob/master/lib/rate-limiter.interceptor.ts#L169)

**Comment by csulit** [comment]
> will be fixed on next version with adding an optional chaining operator at [here](https://github.com/ozkanonur/nestjs-rate-limiter/blob/master/lib/rate-limiter.interceptor.ts#L169)

Any workaround?

**Comment by ozkanonur** [comment]
It fixed on v2.7.2, you will see no longer that error. But the point is, if your request context doesn't have an IP in it, the rate limiter won't work.
